library/mageia

```diff
@@ -1,14 +1,14 @@
 Maintainers: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
 GitRepo: https://github.com/juanluisbaptiste/docker-brew-mageia.git
-GitCommit: 8fa1a171030b6bc20216cc6a6d3f6d77662eabc6
+GitCommit: 718477bf0e9298cf8ed9f08c5229713356e4728d
 GitFetch: refs/heads/dist
 
-Tags: 6, latest
+Tags: 6,
```

Comment:
You've got a trailing comma here:

```suggestion
Tags: 6
```

Reply:
Ahhhhhhhhhhhhhh I could not understand what was happening with that simple change ! thanks for the catch.